frontend/src/utils/axiosHeaders.js

```diff
@@ -10,7 +10,6 @@ export const validHeader = (params) => ({
 export const multipartFileHeader = (params) => ({
   headers: {
     ...authHeader(),
-    'Content-Type': 'multipart/form-data'
```

Comment:
Jesteś pewien że usunięcie tego jest od tak bezkarnie możliwe ? Bo gdy robiłem to z Adim to ta linijka była konieczna. Niby po stronie frontu się nie sypie ale sprawdź może czy na backu wszystko działa jak należy

Reply:
Gdy prezentowałeś dzisiaj to zauważyłem, że chyba dane w tym csv miały jakieś chińskie znaczki czy coś (no krzaki zamiast pl znaków). Sprawdź czy przypadkiem nie przez to. A przede wszystkim czy mi się nie przywidziało

Reply:
To moje formatowanie ustawione dziwnie na kompie xd

Sprawdziłem w docsach i odpala się normalnie

Reply:
Odnośnie backu - wydaje mi się że działa dobrze nawet bez tej linijki - pliki nie są "ucinane", i można je normalnie pobrać, więc najwyraźniej nie jest potrzebne bo to inferuje back sam... a przynajmniej tak mi się wydaje

Reply:
@ajablonsk1 Pamiętasz może dlaczego to było wtedy takie ważne ?

Reply:
Dodam tylko że nawet bez nagłówka żądanie jest w postaci form-data:
![firefox_Sr7ftcGuII](https://user-images.githubusercontent.com/56168780/185745747-5a3a1d13-3526-4030-96a0-44d0657998c3.png)

Wydaje mi się że axios inferuje ten typ na podstawie tego że dostaje w body formkę i sam doczepia `Content-Type` w tym przypadku, ale mogę się mylić

Reply:
ewentualnie - to jak jest zbudowany obiekt `FormData` nawet po zwykłym wrzuceniu w body jest prawidłowo parsowany przez springa, ale w obu przypadkach strzelam z przypuszczeniami

Reply:
Moim zdaniem najbardziej prawdopodobna wersja wydarzeń jest taka że rest api ogarnia że to multipart data z automatu bo podajemy form data. Temat chyba do zamknięcia